samples/metrics_monitor/src/cttmetrics_i915_pmu.cpp

```diff
@@ -268,11 +311,11 @@ extern "C"
 cttStatus CTTMetrics_PMU_Init()
 {
     int configs[] = {
-        I915_PMU_ENGINE_BUSY(I915_ENGINE_CLASS_RENDER, 0),
-        I915_PMU_ENGINE_BUSY(I915_ENGINE_CLASS_VIDEO, 0),
-        I915_PMU_ENGINE_BUSY(I915_ENGINE_CLASS_VIDEO, 1),
-        I915_PMU_ENGINE_BUSY(I915_ENGINE_CLASS_COPY, 0),
-        I915_PMU_ENGINE_BUSY(I915_ENGINE_CLASS_VIDEO_ENHANCE, 0),
+        I915_PMU_ENGINE_BUSY(I915_ENGINE_CLASS_RENDER, 0), /*I915_PMU_ENGINE_WAIT(I915_ENGINE_CLASS_RENDER, 0), I915_PMU_ENGINE_SEMA(I915_ENGINE_CLASS_RENDER, 0),*/
```

Comment:
Add comments that user can't mix metrics from different groups.

int configs[] = {
 // Render engined metrics
...
 // VDBOX engined metrics
...
 // global metrics
...
};